deploy-chroot/create.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -e -u
+
+cd "$(dirname "$0")"
+
+if [ "${HOMEWORLD_DEPLOY_CHROOT:-}" = "" -o ! -e "$(dirname HOMEWORLD_DEPLOY_CHROOT)" ]
+then
+    echo "invalid path to chroot: ${HOMEWORLD_DEPLOY_CHROOT:-}" 1>&2
+    echo '(have you populated $HOMEWORLD_DEPLOY_CHROOT?)'
+    echo '(try export HOMEWORLD_DEPLOY_CHROOT=$HOME/chroot)'
+    exit 1
+fi
+
+if [[ "${HOMEWORLD_DEPLOY_CHROOT}" =~ " " ]]
+then
+    echo "chroot name cannot include a space" 1>&2
+    exit 1
+fi
+
+if [ -e "${HOMEWORLD_DEPLOY_CHROOT}" ]
+then
+    echo "chroot already exists" 1>&2
+    exit 1
+fi
+
+if [ "$USER" = "root" ]
+then
+    echo "this script should be run as a user with sudo capabilities" 1>&2
+    exit 1
+fi
+
+mkdir -m 'u=rwx,go=rx' "${HOMEWORLD_DEPLOY_CHROOT}"
+# todo: separate chroot-packages; install bridge-utils; qemu-utils; linux-image-amd64; qemu-system-x86; whichever gives ssh-agent
```

Comment:
Any reason not to do this now, if we need it anyway?

Reply:
oops, that todo is todone